tests/site_nav.js

```diff
@@ -79,8 +75,8 @@ describe('SiteNav', function() {
       expect(this.$subLists.last().attr('aria-hidden')).to.equal('true');
     });
 
-    it('should assign aria-haspopup to all links to the sub lists', function() {
-      var $link = this.$fixture.find('#site-menu .js-nav-drop-link');
+    it('should assign aria-haspopup to all toggle buttons for the sub lists', function() {
+      var $link = this.$fixture.find('.js-sublist-toggle');
```

Comment:
This test doesn't seem to do exactly what the description says--we're asserting that there's at least one `.js-sublist-toggle`, and that the first one has the "aria-haspopup" attribute. Which is fine if there's exactly one toggle button. But if not, we might use something like the following:

```javascript
var $toggles = this.$fixture.find('.js-sublist-toggle');
var $popupToggles = this.$fixture.find('.js-sublist-toggle[aria-haspopup="true"]');
expect($toggles.length).to.be.ok;
expect($toggles.length).to.equal($popupToggles.length);
```